modules/cucumber-reportportal-formatter.js

```diff
@@ -3,6 +3,12 @@ const ReportPortalClient = require('reportportal-client');
 const Path = require('path');
 const pjson = require('../package.json');
 
+const formatCodeRef = (path, itemName) => {
+  const codeRef = path.replace(/\\/g, '/');
```

Comment:
Will this separator `/\\/g` be in other systems, in Linux, for example?

Reply:
As I know `/` used in Unix systems, so we just need to replace `\`.

Reply:
It won't be. Separator in POSIX system is `/`, as required. So this regex just for Windows `/` separator.

Reply:
http://www.mit.edu/~6.005/fa09/resources/avoid-dependent-code.html